ngraph/core/reference/include/ngraph/runtime/reference/detection_output.hpp

```diff
@@ -562,7 +562,10 @@ namespace ngraph
                          const dataType* _armLocation,
                          dataType* result)
                 {
-                    std::memset(result, 0, outTotalSize * sizeof(dataType));
+                    for (size_t i = 0; i < outTotalSize; i++)
+                    {
+                        result[i] = dataType(0);
+                    }
```

Comment:
why?

Reply:
For float16 (bfloat16) types. I get the next error message:
```
memset(void*, int, size_t)’ clearing an object of non-trivial type use assignment or value-initialization instead
```